docs/design/rosbag2_record_replay_action.md

```diff
@@ -128,12 +128,12 @@ Newly added arguments:
 | :-- | :-- |
 | --actions action [action ...] | Space-delimited list of actions to play. |
 | --exclude-actions action [action ...] | Space-delimited list of actions not to play. |
-| --send-action-goal | Send recorded action goal instead of recorded service events and action internal topics. |
+| --send-actions-as-client | Send the send_goal request, cancel_goal request, and get_result request respectively based on the recorded send_goal, cancel_goal, and get_result event messages. |
```

Comment:
Also, we need to add a notice that messages on action's "status topic" [NameSpace/]ActionName/_action/status
and "feedback topic" [NameSpace/]ActionName/_action/feedback will not be sent because they are expected to be sent from the server side.

Reply:
Okay. I will add this note.

Reply:
@MichaelOrlov 

Update completed at  https://github.com/ros2/rosbag2/pull/1928/commits/3f1ead3da6b8a4d4c4dcfd82f4753a5e369e32ae

Reply:
Thank you!